std/file.d

```diff
@@ -2325,6 +2350,15 @@ else version(Posix) string readLink(C)(const(C)[] link) @safe
     throw new FileException(to!string(link), "Path is too long to read.");
 }
 
+/// ditto
+version(StdDdoc) string readLink(R)(R link)
```

Comment:
So `readLink` uses an overload to deal with this, while `symlink` deals with the same problem using just one function. Why the disparity?